src/main/java/org/opentripplanner/apis/support/TileJson.java

```diff
@@ -72,4 +57,42 @@ public TileJson(
     var c = envelope.center();
     center = new double[] { c.longitude(), c.latitude(), 9 };
   }
+
+  /**
+   * Creates a vector source layer URL from a hard-coded path plus information from the incoming
+   * HTTP request.
+   */
+  public static String urlWithDefaultPath(
+    UriInfo uri,
+    HttpHeaders headers,
+    String layers,
+    String ignoreRouterId,
```

Comment:
I know this isn't the only place where we use this param but I feel like the type/value doesn't match the name.

Reply:
You're probably talking about the `layers` parameter, isn't it?

Reply:
I've refactored it here: https://github.com/opentripplanner/OpenTripPlanner/pull/5627/commits/cb59e3dd2d8982a2efde8d810322a3057910dcf0

Reply:
I meant `ignoreRouterId` but the layers are better as a list also

Reply:
What type should the routerId have?

Reply:
routerId should have a string type but ignoreRouterId should be a boolean/function based on the name.